MoreLinq.Test/PreScanTest.cs

```diff
@@ -22,6 +22,29 @@ namespace MoreLinq.Test
     [TestFixture]
     public class PreScanTest
     {
+        [Test]
+        public void PreScanIsLazy()
+        {
+            new BreakingSequence<int>().PreScan(BreakingFunc.Of<int, int, int>(), 0);
+        }
+
+        [Test]
+        public void PreScanWithEmptySequence()
+        {
+            var source = Enumerable.Empty<int>();
+            var result = source.PreScan(BreakingFunc.Of<int, int, int>(), 0);
+
+            Assert.IsFalse(result.Any());
```

Comment:
This assertion is more clearly & explicitly expressed as: `Assert.That(result, Is.Empty);`

In case of failure, it will say something like this:

```
  Expected: <empty>
  But was:  < 2 >
```

instead of the following, which all `Assert.IsFalse` can say:

```
  Expected: False
  But was:  True
```